openmdao/docs/developer_docs/repo_guide/travis.rst

```diff
@@ -0,0 +1,208 @@
+        - ubuntu-toolchain-r-test
+        packages:
+        - gfortran
+        - libblas-dev
+        - liblapack-dev
+        - libopenmpi-dev
+        - openmpi-bin
+
+    before_install:
+    - if [ "$PY" = "2.7" ];  then wget "https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh" -O miniconda.sh; fi
+    - if [ "$PY" = "3.6" ];  then wget "https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh" -O miniconda.sh; fi
+    - chmod +x miniconda.sh;
+    - ./miniconda.sh -b  -p /home/travis/miniconda;
+    - export PATH=/home/travis/miniconda/bin:$PATH;
+    - if  [ "$TRAVIS_REPO_SLUG" = "OpenMDAO/dymos" ] && [ "$TRAVIS_PULL_REQUEST" = "false" ]; then
+        MASTER_BUILD=1;
+      fi
+
+    install:
+    - conda install --yes python=$PY numpy scipy nose sphinx mock swig pip;
+    - pip install --upgrade pip
+    - sudo apt-get install gfortran
+    - pip install numpy==1.14.1
+    - pip install scipy==1.0.0
+    - pip install mpi4py
+    - pip install matplotlib
+    - pip install nose
+    - pip install networkx
+    - pip install testflo
+    - pip install pyyaml
+    - pip install coveralls
+    - pip install --user travis-sphinx;
+
+    # install pyoptsparse
+    - git clone https://github.com/OpenMDAO/pyoptsparse.git;
+    - cd pyoptsparse;
+    - python setup.py install;
+    - cd ..;
+
+    # install MBI
+    - git clone https://github.com/OpenMDAO/MBI.git;
+    - cd MBI;
+    - python setup.py build install;
+    - cd ..;
+
+    # install OpenMDAO in developer mode so we have access to its sphinx extensions
+    - git clone https://github.com/OpenMDAO/OpenMDAO.git;
+    - cd OpenMDAO;
+    - pip install -e .;
+    - cd ..;
+
+    # install your project itself in developer mode.
+    - pip install -e .;
+
+    script:
+    - testflo -n 1 your_project --pre_announce --coverage --coverpkg dymos;
+    - travis-sphinx build --source=your_project/docs;
+
+    after_success:
+    - if [ "$MASTER_BUILD" ] && [ "$UPLOAD_DOCS" ]; then
+        travis-sphinx deploy;
+      fi
+    - coveralls;
+
+Coverage
+--------
+
+Coverage is a tool that shows developers how many lines of their code that are being executed by your current test suite.
+
+To use this tool, set up an account at the website `coveralls.io <https://coveralls.io>`_, login using your Github credentials,
+and then activate the appropriate github repo for coverage results. This sets up the site to receive results. The other side
+of the equation is taken care of in your .travis.yml file, as I will show here using OpenMDAO's example:
+
+.. code-block:: python
+
+    script:
+        testflo -n 1 openmdao --coverage  --coverpkg openmdao --cover-omit \*tests/\*  --cover-omit \*devtools/\* --cover-omit \*test_suite/\* --cover-omit \*docs/\*;
+
+    after_success:
+        coveralls
+
+The point of the above example is to use testflo to run things, and set the `coverpkg` to your project, and the `cover-omit` dirs to exclude from coverage, use coveralls
```

Comment:
suggest: "run things" -> "run the test suite"
suggest rewording to explain that testflo generates coverage data (via the coverpkg and cover-omit options) and pointing to the testflo github for further information.  (maybe we should add a dump of `testflo --help` to the README there or make a quick wiki page showing the options)

Reply:
second thought.. add a project description (e.g. just `testflo --help`) to the testflo page on pypi and link to that

Reply:
made a link and added a story to fix the testflo page on pypi